docs/dev/sstable-scylla-format.md

```diff
@@ -55,6 +56,10 @@ Scylla executable that created the sstable.
 `scylla_version` (tag 8): a string containing the version of the
 Scylla executable that created the sstable.
 
+`clustering_position_metadata` (tag 9): stores the very first and very last clustering
+keys in this sstable. This is used, when working with SSTable runs, which had their
```

Comment:
nit: don't break long lines.
.md is formatted into html so the browser better do it.

Reply:
counter-nit: you can break long lines because the renderer works in terms of paragraphs. "Soft" line-breaks such as the ones we use to break long lines don't appear when MD is rendered.

You can test this by opening this document in your browser (it already has broken long lines): 
https://github.com/scylladb/scylladb/blob/master/docs/dev/sstable-scylla-format.md

In other words, I propose the bike-shed be painted blue :P

Reply:
I agree it makes sense to break lines in a paragraph on sentence or separator (comma / semi-column) boundaries for clarity, but not on artificial "80 column width" lines.